docs/sources/articles/b2d_volume_resize.md

```diff
@@ -6,21 +6,40 @@ page_keywords: boot2docker, volume, virtualbox
 
 If you're using Boot2Docker with a large number of images, or the images you're
 working with are very large, your pulls might start failing with "no space left 
-on device" errors when the Boot2Docker volume fills up. The solution is to 
-increase the volume size by first cloning it, then resizing it using a disk 
+on device" errors when the Boot2Docker volume fills up. There are two solutions
+in this case:
+
+## Solution 1: Fast, but it will delete all local docker data
+
+Add the following lines to $HOME/.boot2docker/profile (creating it if it’s not
+already there):
+
+    # Disk image size in MB
+    DiskSize = 50000
```

Comment:
There's nothing magic about this allocation, right? It could be whatever is needed?